src/intro.md

```diff
@@ -6,46 +6,43 @@ The Internet currently lacks effective, privacy-preserving **Data Provenance**.
+The TLSNotary protocol allows the `Prover` to selectively prove the authenticity of arbitrary parts of the data to a `Verifier`. In this **selective disclosure** phase, the `Prover` can **redact** sensitive information from the data prior to sharing it with the `Verifier`.
 
-### ③ Selective Disclosure
+This capability can be paired with Zero-Knowledge Proofs to prove properties of the redacted data without revealing the data itself.
 
-The TLSNotary protocol provides a means for the `User` to selectively prove the authenticity of arbitrary sections of the transcript to a `Verifier`. In this context, the `User` is often referred to as the `Prover`.
+### ③ Data Verification
 
-The `User` can **redact sections** of the transcript, thereby not disclosing sensitive data to the `Verifier`. This capability can be paired with Zero-Knowledge Proofs to prove properties of the redacted data without revealing the data itself.
+The `Verifier` now validates the proof received from the `Prover`. The data origin can be verified by inspecting the `Server` certificate through trusted certificate authorities (CAs). The `Verifier` can now make assertions about the non-redacted content of the transcript.
 
-### ④ Verification
+## TLS verification with a general-purpose Notary
 
-A `Verifier` validates the proof received from the `User`. By comparing the signature against the `Notary`'s public key, the `Verifier` ensures that the `User` did not tamper with the data.  
-The data origin can be verified by inspecting the `Server` certificate through trusted certificate authorities (CAs).
+Since the validation of the TLS traffic neither reveals anything about the plain text of the TLS session nor about the `Server`, it is possible to outsource the MPC-TLS verification to a general-purpose TLS verifier ①, which we term a `Notary`. This `Notary` can sign (aka *notarize*) ② the data, making it portable. The `Prover` can then take this signed data and selectively disclose ③ sections to an application-specific verifier, which then verifies the data ④.
+
+![](./png-diagrams/overview3.png)
 
-### Trust Assumptions
+In this setup, the `Notary` cryptographically signs commitments to the data and the server's identity. The `Prover` can store this signed data, redact it, and share it with any data `Verifier` as they see fit, making the signed data both reusable and portable.
 
-TLSNotary requires a trust assumption. A `Verifier` of a proof must trust that the `Notary` did not collude with the `User` to forge it. This trust can be minimized by requiring multiple proofs, each signed by a different notary.
+Data verifiers will only accept the signed data if they deem the `Notary` trustworthy. A verifier can also require signed data from multiple Notaries to rule out collusion between the `Prover` and a `Notary`.
```

Comment:
```suggestion
Data `Verifiers` will only accept the signed data if they trust the `Notary`. A `Verifier` can also require signed data from multiple `Notaries` to rule out collusion between the `Prover` and a `Notary`.
```

Reply:
I did not use back ticks here because this is about a different Verifier than earlier in the text: it only verifies the data, not the TLS transcript.

Maybe we have to come up with clearer names, to avoid confusion here.

Reply:
Maybe it makes sense to introduce the term `Data Verifier` in this section and use the back ticks accordingly?

Reply:
04abaf5defa9b00a4aaa3de15b88329d184d50df

Reply:
ok, makes sense not use ticks around verifier here.